components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4312,4 +4347,11 @@ public static String resolveTenantDomain(HttpServletRequest request) {
         }
         return IdentityTenantUtil.getTenantDomainFromContext();
     }
+
+    public static void log(String componentId, Map<String, Object> input, String resultStatus, String resultMessage,
+                            String actionId, Map<String, Object> configurations) {
+
+        LoggerUtils.triggerDiagnosticLogEvent(componentId, input, resultStatus, resultMessage, actionId,
```

Comment:
Will be have different componentId using this util other than `"oauth-inbound-service"` ? in that case, we can remove it from parameter and define it here

Reply:
No, all the logs are having `"oauth-inbound-service"` as the componentId. +1. Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1737/commits/ba650f85362ebaa25fbeec0b1716bccb3e5aaced